proteus-orchestrate/orchestrate/handler/rendezvous.go

```
@@ -73,34 +54,37 @@ func GetCollectors(db *sqlx.DB) (Collectors, error) {
 		err = rows.Scan(&ctype, &caddress, &cfront)
 		if err != nil {
 			ctx.WithError(err).Error("failed to get collector row")
-			continue
+			// XXX should we fail fast? I think so
```

Like panic?

---

I think panic is too extreme (as it leads to the software crashing), I just noted this in a comment that we should ensure it makes indeed sense to throw an error instead of just skipping the item like it was done before.

---

Maybe the comment should say, "Is it good that we are failing fast? I think so"